lib/view_component/test_helpers.rb

```diff
@@ -190,6 +190,29 @@ def with_request_url(path, host: nil)
       @vc_test_controller = old_controller
     end
 
+    # Set the controller helpers to be used while executing the given block,
+    #
+    # ```ruby
+    # with_helpers(UsersHelper) do
+    #   render_inline(MyComponent.new)
+    # end
+    # ```
+    #
+    # @param *args [Array<Module>] The helpers to be used.
+    def with_helpers(*args)
+      args.each do |helper_mthd|
+        vc_test_controller.class_eval(<<-RUBY, __FILE__, __LINE__ + 1)
+          def #{helper_mthd}
+            return #{return send(helper_mthd)}
```

Comment:
Not sure why we need the extra `return` here?

```suggestion
            return #{send(helper_mthd)}
```

Reply:
@Spone , thank you for the spot, but that was my initail implementation but it always gave this error

```
Error:
RenderingTest#test_use_helper:
ArgumentError: wrong number of arguments (given 0, expected 1)
    /home/reegan/view_component/lib/view_component/test_helpers.rb:206:in `method'
    /home/reegan/view_component/lib/view_component/test_helpers.rb:206:in `test_helper'
    /home/reegan/view_component/lib/view_component/test_helpers.rb:212:in `test_helper'
    /home/reegan/view_component/lib/view_component/use_helpers.rb:12:in `test_helper'
    /home/reegan/view_component/test/sandbox/app/components/use_helpers_component.html.erb:2:in `call'
    /home/reegan/view_component/lib/view_component/compiler.rb:120:in `render_template_for'
    /home/reegan/view_component/lib/view_component/base.rb:107:in `render_in'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/actionview-7.0.4.3/lib/action_view/helpers/rendering_helper.rb:42:in `render'
    /home/reegan/view_component/lib/view_component/test_helpers.rb:50:in `render_inline'
    /home/reegan/view_component/test/sandbox/test/rendering_test.rb:1100:in `block in test_use_helper'
    /home/reegan/view_component/lib/view_component/test_helpers.rb:213:in `with_helpers'
    /home/reegan/view_component/test/sandbox/test/rendering_test.rb:1099:in `test_use_helper'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest/test.rb:102:in `block (3 levels) in run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest/test.rb:199:in `capture_exceptions'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest/test.rb:97:in `block (2 levels) in run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:296:in `time_it'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest/test.rb:96:in `block in run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:391:in `on_signal'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest/test.rb:247:in `with_info_handler'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest/test.rb:95:in `run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:1051:in `run_one_method'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:365:in `run_one_method'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:352:in `block (2 levels) in run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:351:in `each'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:351:in `block in run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:391:in `on_signal'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:378:in `with_info_handler'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:350:in `run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:182:in `block in __run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:182:in `map'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:182:in `__run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:159:in `run'
    /home/reegan/.asdf/installs/ruby/3.0.3/lib/ruby/gems/3.0.0/gems/minitest-5.18.0/lib/minitest.rb:83:in `block in autorun'
```

If you maybe have some context into this issue it would be much appriated